cmd/satellite/main.go

```diff
@@ -63,50 +63,58 @@ func main() {
 		}
 	}()
 
-	go func() {
-		backlog, _ := recoverState(satellite_configuration.Satellite.Cache)
+	backlog, _ := recoverState(satellite_configuration.Satellite.Cache)
 
-		for stat := range backlog {
-			err := s.sendGPUStatus(backlog[stat])
+	for stat := range backlog {
+		err := s.sendGPUStatus(backlog[stat])
 
-			if err != nil {
-				log.Error("Failed to send backlogged GPU stat message", "err", err)
-			}
+		if err != nil {
+			log.Error("Failed to send backlogged GPU stat message", "err", err)
 		}
+	}
 
-		backlog = make([][]uplink.GPUStatSample, 0)
+	backlog = make([][]uplink.GPUStatSample, 0)
 
-		collectGPUStatTicker := time.NewTicker(time.Duration(satellite_configuration.Satellite.DataInterval) * time.Second)
-		publishGPUStatTicker := time.NewTicker(time.Duration(satellite_configuration.Satellite.DataInterval) * time.Second)
+	collectGPUStatTicker := time.NewTicker(time.Duration(satellite_configuration.Satellite.DataInterval))
```

Comment:
If we remove `* time.Second` here, does that mean that our configured satellite data & heartbeat intervals have to be in milliseconds? I think this is more precise than we need to be (ie. I'd rather change the heartbeat interval to be in seconds rather than change the tickers to not be in seconds)

Reply:
Yeah, that's a seperate problem, I'm working on it